online-shop/src/Styles/ComponentsStyles/Navbar.scss

```diff
@@ -0,0 +1,7 @@
+@charset "utf-8";
+
+img {
+    object-fit: cover;
+}
+
+@import '../Chapter4Bulma/App.sass';
```

Comment:
A small refactoring of the folder structure would be nice :) It should not reflect the chapters in this training, but the purpose of the files contained (e.g. `styles`).